cypher/cypher-docs/src/docs/dev/ql/administration/security/dbms/all-management-syntax.asciidoc

```diff
@@ -1,7 +0,0 @@
-.Command syntax
```

Comment:
Content was moved to cypherManual/asciidoc/access-control/dbms-administration.adoc

https://neo4j.com/docs/cypher-manual/4.3/access-control/dbms-administration/#access-control-dbms-administration-all